octoprint_discordremote/command.py

```diff
@@ -53,8 +53,13 @@ def parse_command(self, string, user=None):
         parts = re.split(r'\s+', string)
 
         command_string = "help"
-        if len(parts[0]) > len(prefix_str) and prefix_str == parts[0][:prefix_len]:
+        has_command_prefix = False
+        if len(parts[0]) >= prefix_len and prefix_str == parts[0][:prefix_len]:
```

Comment:
I think this could be simplified down to:
```
if len(parts[0]) <= prefix_len or prefix_str != parts[0][:prefix_len]:
    return None, None

command_string = parts[0][prefix_len:]
```

Make that change and I'll merge it.

Reply:
Great, sounds good to me with one slight change: `len(parts[0]) < prefix_len` instead of `len(parts[0]) <= prefix_len` so that if the command prefix alone is sent, then they'll still get help.

Reply:
Sounds good. Just noticed the command_string="help" is a bit pointless,but i'll merge and fix it.
Thank you for your contribution.

Reply:
Thanks!